### DISCUSSION TITLE
MOST STABLE  SOCKET.IO CLIENT VERSION IS THE 4.0.X SERIES

### DISCUSSION THREAD

**Discussion by khristian121** [miguelgrinberg/python-socketio#1110] [Q&A] [answered]
HAPPY NEW YEAR
PLEASE IF I MAY ASK; WHAT IS THE MOST STABLE SOCKET.IO CLIENT VERSION OF THE 4.0.X SERIES WITHOUT ANY PROBLEMS

**Comment by miguelgrinberg** [comment, marked as answer]
The latest one.

**Reply by khristian121** [reply]
THAT'S 4.6.1 INNIT
I APPRECIATE
